plugin-annotation/src/test/java/com/mapbox/maps/plugin/annotation/AnnotationPluginImplTest.kt

```diff
@@ -65,8 +65,9 @@ class AnnotationPluginImplTest {
 
     annotationPluginImpl.removeAnnotationManager(circleAnnotationManager)
     annotationPluginImpl.removeAnnotationManager(pointAnnotationManager)
-    annotationPluginImpl.removeAnnotationManager(polygonAnnotationManager)
-    annotationPluginImpl.removeAnnotationManager(polylineAnnotationManager)
+    assertEquals(2, annotationPluginImpl.managerList.size)
+
```

Comment:
nit : empty line

Reply:
This is intentional as this block to verify 2 annotationManagers are removed